### PR TITLE
[MNO-98] Display App Pricing

### PR DESCRIPTION
**Pull request by winnietan**

### CODE REVIEW

File: src/app/views/marketplace/marketplace-app.html [hunk 1]
```diff
@@ -106,6 +106,29 @@ <h2 class="media-heading">{{vm.app.name}}</h2>
           </div>
         </div>
 
+      <!-- Pricing -->
+      <div ng-show="vm.isPriceShown" class="row top-buffer-1">
```
Comment (alexnoox) [inline src/app/views/marketplace/marketplace-app.html]: A `ng-if` would be more adapted as this section will never be shown during the session if it is not enabled.

Comment (alexnoox) [inline src/app/views/marketplace/marketplace-app.html]: @winnietan I forgot to mention you ;)

Comment (winnietan) [inline src/app/views/marketplace/marketplace-app.html]: ok, I will try to change it. :)

**Comment by alexnoox** [comment]
@winnietan Perfect thx

**Pull request merged**